agnostic-angular/copystyles.js

```diff
@@ -165,18 +165,29 @@ fs.writeFileSync('./libs/ag/src/lib/loader.css', css, 'utf8');
 /**
  * Menu
  */
- css = fs.readFileSync('../agnostic-css/src/components/menu/menu.css', 'utf8');
- fs.writeFileSync('./libs/ag/src/lib/menu.css', css, 'utf8');
- 
- /**
-  * Menu Item
-  */
- css = fs.readFileSync(
-   '../agnostic-css/src/components/menu/menu-item.css',
-   'utf8'
- );
- fs.writeFileSync('./libs/ag/src/lib/menu-item.css', css, 'utf8');
- 
+css = fs.readFileSync('../agnostic-css/src/components/menu/menu.css', 'utf8');
+fs.writeFileSync('./libs/ag/src/lib/menu.css', css, 'utf8');
+
+/**
+ * Menu Item
+ */
+css = fs.readFileSync(
+  '../agnostic-css/src/components/menu/menu-item.css',
+  'utf8'
+);
+fs.writeFileSync('./libs/ag/src/lib/menu-item.css', css, 'utf8');
+
+ // TODO -- MenuTrigger
```

Comment:
This is a TODO because this CSS is yet to be used. My thought is we'll add `styleUrls: [..., './menu-trigger.css']` which is now available.

I'm not sure where it's easiest to put these "menu trigger button" given the `:host` element when you have sub-components. @vitale232 if you have idiomatic idea of how to do this I will trust you're Angular sensibilities 😄